Detectors/MUON/MCH/Base/src/Digit.cxx

```diff
@@ -19,7 +19,13 @@ bool closeEnough(double x, double y, double eps = 1E-6)
   return std::fabs(x - y) <= eps * std::max(1.0, std::max(std::fabs(x), std::fabs(y)));
 }
 
-Digit::Digit(double time, int detid, int pad, unsigned long adc)
+Digit::Digit(int sampaTime, int detid, int pad, unsigned long adc)
```

Comment:
while we're at changing the Digit ctor, I would go a step further and move the time as the last parameter (and with a default value of 0 ?)

Reply:
I have moved the time parameter at the end, and removed the other constructor that takes a double value.
As a consequence, I had to modify quite some files in the encoder part, and it would be good if you could have a closer look.